managed/src/main/java/com/yugabyte/yw/cloud/aws/AWSCloudImpl.java

```diff
@@ -33,6 +35,24 @@ public AmazonEC2 getEcC2Client(Provider provider, Region r) {
       .build();
   }
 
+  @Override
+  public boolean isAWSValidCredentials(Map<String, String> config) {
+    BasicAWSCredentials awsCreds = new BasicAWSCredentials(config.get("AWS_ACCESS_KEY_ID"),
+      config.get("AWS_SECRET_ACCESS_KEY"));
+    try {
+      AmazonEC2 ec2Client = AmazonEC2ClientBuilder
+        .standard()
+        .withCredentials(new AWSStaticCredentialsProvider(awsCreds))
+        .withRegion(Regions.US_EAST_1)
+        .build();
+      DescribeRegionsResult regions_response = ec2Client.describeRegions();
+      return true;
+    } catch (Exception e) {
+      return false;
```

Comment:
What if we get an exception not related to incorrect credentials?

Reply:
I am assuming that won't be the case here. Could you please point me to such scenarios? Anyway, if we get any exceptions for whatever reasons;that means we are unable to validate the credentials and we won't  let user to proceed(This will be very rare).

cc: @sb-yb

Reply:
Simple example, I think it could happen for any cloud, - connection timeout (AWS network is unreachable or something like this). So, at least, you should log the exception (reason why the credentials are failed). But logging the exception be careful - the used credentials should not go into logs (otherwise it is a security hole).

Reply:
Sounds good! Will log a exception by considering security hole. Thanks!

Reply:
Done

Reply:
This log.Error message won't contain provided credentials so we are good at that.

Reply:
I think here you should use dry run. 
See https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/ec2/AmazonEC2Client.html#dryRun-com.amazonaws.services.ec2.model.DryRunSupportedRequest-